util/__init__.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+# because we want a round function which does the same on Python 2.7 and 3.x
+from numpy import around as round
+
+
+def format_float_with_standard_uncertainty(value, standard_uncertainty, minimum=1e-12):
+    """
+    Formats a float, including the uncertainty in its value.
+
+    Parameters
+    ----------
+    value : float
+    standard_uncertainty : float
+    minimum : float
+
+    Returns
+    -------
+    str
+
+    Examples
+    --------
+    >>> format_float_with_standard_uncertainty(5e-3, 1e-3)
+    '0.0050(10)'
+    >>> format_float_with_standard_uncertainty(5e-3, 1e-6)
+    '0.0050000(10)'
+    """
+    if standard_uncertainty <= minimum:
+        dp = -int(math.log10(minimum))
+        return str(round(value, dp))
+    precision = -int(round(math.log10(standard_uncertainty)))
+    if precision > -1:
+        su = standard_uncertainty * math.pow(10, precision)
+        if round(su, 1) < 2:
+            su *= 10
+            precision += 1
+        fmt_str = "%%.%if(%%i)" % precision
+        return fmt_str % (value, round(su))
```

Comment:
```suggestion
        return "{value:.{precision}f}({irsu})".format(
            value=value, precision=precision, irsu=int(round(su)),
        )
```
possibly less obscure as it avoids the double string formatting `"%%..%..%%" % (...) % (...)`